### ISSUE TITLE
In models from QueryObject method columns are organized alphabetically 

### ISSUE THREAD

**Issue by wondie** [gltn/stdm#69]
Data with key - value from models are sorted alphabetically. This is not desirable for display purpose. For instance, last name comes before first name. In the previous configuration, it was possible to get columns ordered as they were created, which gives the choice of ordering columns to the user. This problem is witnessed in the treewidget of New STR Wizard. Will be seen in other modules using the method. Is it possible to order them based on the order set in the configuration.

**Comment by gkahiu** [comment]
There is no control with regard to how values are added in the query object. That said, it should be easy to get the column values they way they were created in the configuration. For instance, the following pseudo-code should do produce the desired results:
```
for c in entity.columns.keys():
     if hasattr(result_obj, c):
         col_value = getattr(result_obj, c)
         display = u'[Column Display]: {0}'.format(col_value)
```

Does this address your issue?

**Comment by wondie** [comment]
Thank you for the code. It addresses the issue. However, I just found out there is a method called utils.set_creation_order in alchemy which seems to be used for the same purpose. Could we implement it in  QueryObject method to prevent the alphabetical sorting?

**Comment by gkahiu** [comment]
I have never seen it before and cannot seem to find any proper
documentation for it. If you have used it and it has worked then go ahead
and implement it.

**Comment by wondie** [comment]
It is not easy to get anything about it. When I applied it on STDMDb class, nothing changes. So I removed it. In stead, I found another workaround. Adding column list in queryObject(), 
example:
`
 db_obj.queryObject(entity_display_columns(entity)).filter(db_model.id == sel_id).first()
`
entity_display_columns(entity) lists display columns and it gets them from entity.columns so it is ordered correctly. It is located in utils/util.py
